dissect/evidence/adcrypt/c_adcrypt.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from dissect.cstruct import cstruct
+
+adcrypt_def = """
+#define ADCRYPT_MAGIC ADCRYPT\00
```

Comment:
```suggestion
#define ADCRYPT_MAGIC b"ADCRYPT\00"
```
cstruct lets us cheat a bit.

Reply:
That does not seem to work (`dissect-cstruct==4.7.dev1`).

```python
ipdb> c_adcrypt.ADCRYPT_MAGIC
'b"ADCRYPT\x00"'
```

Reply:
Ah the `\00` seems to throw this off.